src/gui/PasswordGeneratorWidget.cpp

```diff
@@ -196,13 +248,13 @@ void PasswordGeneratorWidget::colorStrengthIndicator(double entropy)
     // Set the color and background based on entropy
     // colors are taking from the KDE breeze palette
     // <https://community.kde.org/KDE_Visual_Design_Group/HIG/Color>
-    if (entropy < 35) {
+    if (entropy < 40) {
```

Comment:
I don't know. I think the current entropy value are almost nice for the current password cracking technology.